src/Symfony/Component/Security/Core/Authorization/Voter/AuthenticatedVoter.php

```diff
@@ -47,10 +47,10 @@ public function vote(TokenInterface $token, mixed $subject, array $attributes):
         $result = VoterInterface::ACCESS_ABSTAIN;
         foreach ($attributes as $attribute) {
             if (null === $attribute || (self::IS_AUTHENTICATED_FULLY !== $attribute
-                    && self::IS_AUTHENTICATED_REMEMBERED !== $attribute
-                    && self::IS_AUTHENTICATED !== $attribute
-                    && self::IS_IMPERSONATOR !== $attribute
-                    && self::IS_REMEMBERED !== $attribute)) {
+                && self::IS_AUTHENTICATED_REMEMBERED !== $attribute
+                && self::IS_AUTHENTICATED !== $attribute
+                && self::IS_IMPERSONATOR !== $attribute
+                && self::IS_REMEMBERED !== $attribute)) {
```

Comment:
for automatic change, it's OK

for visual appealing change, I would go manually like this:
```suggestion
                && self::IS_REMEMBERED !== $attribute)
            ) {
```